src/Types/index.ts

```diff
@@ -0,0 +1,18 @@
+export type leaveType = {
```

Comment:
```suggestion
export type Leaf = {
```

Ne suffixe pas par "Type", et au singulier c'est "leaf", pas "leave". Et les types ça commence par une majuscule